src/RpcClient.ts

```diff
@@ -1060,4 +1062,48 @@ export class RpcClient {
       throw new Error(`Error in getTokenAccounts: ${error}`);
     }
   }
-}
+
+  /**
+   * Send a transaction
+   * @param {Transaction} transaction - The transaction to send
+   * @param {HeliusSendOptions} options - Options for sending the transaction
+   * @returns {Promise<TransactionSignature>} - The transaction signature
+   */
+  async sendTransaction(
+    transaction: Transaction | VersionedTransaction,
+    options: HeliusSendOptions = {
+      skipPreflight: true,
+    }
+  ): Promise<TransactionSignature> {
+    let rawTransaction: string;
+    if (transaction instanceof VersionedTransaction) {
+      rawTransaction = Buffer.from(transaction.serialize()).toString('base64');
+    } else {
+      rawTransaction = transaction.serialize().toString('base64');
+    }
+
+    try {
+      const url = `${this.connection.rpcEndpoint}`;
+      const response = await axios.post(
+        url,
+        {
+          jsonrpc: '2.0',
+          id: this.id,
+          method: 'sendTransaction',
+          params: [rawTransaction, {encoding: 'base64', ...options}],
+        },
+        {
+          headers: { 'Content-Type': 'application/json' },
+        }
+      );
+
+      if (response.data.error) {
```

Comment:
Does this follow the same pattern as web3js?

Reply:
at a high level, yes: throwing an error if there is an error returned in the RPC response
but specifically, I targeted it to match the rest of the helius SDK. it throws an Error object whereas the web3js version throws a `SendTransactionError` defined here: https://solana-labs.github.io/solana-web3.js/classes/SendTransactionError.html